### PR TITLE
Version bump for v0.5.0

### PR DESCRIPTION
**Pull request by dnfield**

### CODE REVIEW

**Comment by ened** [comment]
LGTM

File: packages/e2e/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,13 @@
+## 0.5.0
+
+* **Breaking change** by default, tests will use the device window size.
+  Tests can still override the window size by using the `setSurfaceSize` method.
+* **Breaking change** If using Flutter 1.19.0-2.0.pre.196 or greater, the
+  `testTextInput` will no longer automatically register.
+* **Breaking change** If using Flutter 1.19.0-2.0.pre.196 or greater, the
+  `HttpOverrides` will no longer be set by default.
+* Minor formatting changes to Dart code.
```
Comment (digiter) [inline packages/e2e/CHANGELOG.md:9]: Are these release notes relevant to this change?

Comment (dnfield) [reply packages/e2e/CHANGELOG.md:9]: these are the reasons for the patch version bump

**Comment by dnfield** [comment]
I'm going to go ahead and publish this while we wait for the submit queue check.
